README.md

```diff
@@ -1 +1,10 @@
-# Dusk Public Key Infrastructure
+# dusk-pki
+
+![Build Status](https://travis-ci.com/dusk-network/dusk-pki.svg?branch=master)](https://travis-ci.com/dusk-network/dusk-pki)
+![Repository](https://dusk-network.github.io/dusk-pki/repo-badge.svg)](https://github.com/dusk-network/dusk-pki)
+![Documentation](https://dusk-network.github.io/dusk-pki/badge.svg)](https://dusk-network.github.io/dusk-pki/index.html)
```

Comment:
The documentation should be `[![Documentation](https://img.shields.io/badge/docs-pki-blue)](https://docs.rs/dusk-pki/)`